.vscode/settings.json

```diff
@@ -5,5 +5,11 @@
     "**/release-builds": true
   },
   "typescript.validate.enable": false,
-  "javascript.validate.enable": false
+  "javascript.validate.enable": false,
+  "flow.enabled": true,
+  "eslint.enable": true,
+  "prettier.singleQuote": true,
+  "prettier.trailingComma": "es5",
+  "prettier.parser": "flow",
+  "editor.formatOnSave": true
```

Comment:
Yeah, so I do want to be pretty conservative with workspace settings. Because they override user settings, I only want to include stuff that is helpful and desired by 99%+ of people.

So, I really like the other changes (flow, eslint, prettier), since it doesn't make sense to have different values for these. But yeah I think we should remove `formatOnSave`, just because I suspect there are some people who don't like having their code reformated on save, and would prefer to do that manually by running Prettier.

Reply:
I do agree with `formatOnSave` being a bit controversial but the issue (at least for me) is that some projects I’m working on don’t have prettier and so format on save does some really weird stuff so I don’t want it by default. I only want it when there is prettier. 
If we don’t include it in the workspace settings, it means that I will have a unstaged change all the time. 

I don’t think there is a good solution haha

Reply:
ahh that's a good point... Ok, let's turn it on. If other folks complain about it, we can revisit it, but we'll wait until that becomes a real issue.